webpack.config.node.js

```diff
@@ -107,7 +112,9 @@ const webpackConfig = {
 				use: [
 					{
 						loader: 'thread-loader',
-						options: { workers: Math.max( 2, Math.floor( os.cpus().length / 2 ) ) },
+						options: {
+							workers: workerCount,
```

Comment:
do we want the circleci gate here too? maybe just work it into the utility function if we create one?